src/syzygy/syzygy.h

```diff
@@ -20,6 +20,15 @@ enum WdlResult {
 
 namespace {
 
+uint8_t s_dtz {};
+WdlResult s_wdl { WdlResultTableNotActive };
```

Comment:
Declared in header, so just to be safe.

```suggestion
static inline uint8_t s_dtz {};
static inline WdlResult s_wdl { WdlResultTableNotActive };
```

Reply:
Done :)